doc/ref_model/chapters/chapter03.rst

```diff
@@ -5,13 +5,13 @@ It is necessary to clearly define the infrastructure resources and their capabil
 
 The abstraction model presented in this Reference Model (RM) specifies a common set of virtual infrastructure resources that a cloud infrastructure will need to provide to be able to host most of the typical VNF/CNF telco workloads. The intention of this Reference Model is to follow the following principles:
 
--  **Scope:** the model should describe the most relevant virtualised infrastructure resources (incl. acceleration technologies) a cloud infrastructure needs to host Telco workloads
--  **Separation of Concern:** the model should support a clear distinction between the responsibilities related to maintaining the network function virtualisation infrastructure and the responsibilities related to managing the various VNF workloads
--  **Simplicity:** the amount of different types of resources (including their attributes and relationships amongst one another) should be kept to a minimum to reduce the configuration spectrum which needs to be considered
--  **Declarative**: the model should allow for the description of the intended state and configuration of the cloud infrastructure resources for automated life cycle management
--  **Explicit:** the model needs to be rich enough to cover the instantiation and the on-going operation of the cloud infrastructure
--  **Lifecycle:** the model must distinguish between resources which have independent lifecycles but should group together those resources which share a common lifecycle
--  **Aligned:** the model should clearly highlight the dependencies between its components to allow for a well-defined and simplified synchronisation of independent automation tasks.
+- **Scope:** the model should describe the most relevant virtualised infrastructure resources (incl. acceleration technologies) a cloud infrastructure needs to host Telco workloads
```

Comment:
why don't we line wrap to 120 if we touch some lines? (whole contribution)

Reply:
@ulikleber @CsatariGergely I have gone through document and done manual 120 character wrap. However this breaks rendering of bulleted list and so I have wound manual wrapping of these lists.
I am now complete with 120 wrapping to the extent that is does  not introduce other issues.